docs/modules/ROOT/pages/scaling-upgrading.adoc

```diff
@@ -1,6 +1,34 @@
 = Scaling and Upgrading a Hazelcast Deployment
 
-To scale an existing Hazelcast deployment, use the following command:
+To scale an existing Hazelcast deployment, there are two options:
+
+- Modify the yaml definition you used to deploy Hazelcast cluster by updating `clusterSize` field and re-apply it again:
+
+Assuming the following is the existing configuration:
+
+[source,yaml]
+----
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast-sample
+spec:
+  clusterSize: 3
+----
+
+and you want to scale your deployment to 5 replicas, you need to apply the following:
```

Comment:
```suggestion
and you want to scale your deployment to five replicas, you need to apply the following:
```

Reply:
rule of thumb with numbers (excluding measurements and that kind of thing) is one to ten in words, 11 and above in numbers.